website/src/components/Messages/RenderedMarkdown.tsx

```diff
@@ -36,6 +38,7 @@ const sx: SystemStyleObject = {
     bg: "gray.300",
     p: 0.5,
     borderRadius: "2px",
+    marginRight: "40px",
```

Comment:
Also why you need marginRight here?

Reply:
I don't want the button to be hiding some text

Reply:
Maybe show the button when hover only like GitHub does, or maybe add margin to the code element only?

Reply:
also please try `inset-block-end` or `inset-inline-end` instead of `margin-right` for RTL languages.

Reply:
>or maybe add margin to the code element only?

Is this not the right place to put it?

![image](https://user-images.githubusercontent.com/54318514/221344240-a5b051c0-0631-431a-8a96-1b5241950e04.png)

---

>also please try inset-block-end or inset-inline-end instead of margin-right for RTL languages

![image](https://user-images.githubusercontent.com/54318514/221344102-af8ff3a1-8499-400b-ad88-856206f2993a.png)
Would this work instead? The inset-block-end and inset-inline-end don't seem to work for me

![image](https://user-images.githubusercontent.com/54318514/221344005-d2f34769-0c40-4564-a009-72a0649b7ed8.png)
Also, is this how it's supposed to look like?